### PR TITLE
Content

### PR DESCRIPTION
**Pull request by hoptond**
Merges content commits to the master branch

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -195,7 +195,7 @@ header p {
 .contact {
     margin: 0;
     padding: 0;
-    background-color: #222222;
+    background-color: #252826;
```
Comment (Benson-J) [inline style.css:198]: Shows as white in IE10

**Review by Benson-J — COMMENTED** [review]
All good except IE10 colour issue.

File: style.css (hunk 1 above)
Comment (hoptond) [reply style.css:198]: We couldn't resolve this, but decided that it wasn't worth fixing as the site is still usable and readable without it.

**Comment by hoptond** [comment]
On IE10-11, the background for the Projects section is white and there is no way of fixing this without some hideous css selector specifically written for IE. The website is still usable and readable without it.

**Review by mporam — APPROVED** [review]

**Pull request merged**